ChangeLog

```diff
@@ -153,6 +153,14 @@ Release date: TBA
 
   Closes #5261
 
+* Crashes when a list is encountered in a toml configuration do not happen anymore. A new
```

Comment:
This should also be in `2.12.rst`

Reply:
04c0603615d263679b7178a3e9f4aa368d07c525